test/e2e/kubelet_deletes.go

```diff
@@ -0,0 +1,146 @@
+/*
```

Comment:
I wonder whether we don't rather want a test/e2e/kubelet.go to contain all kubelet tests?  Most of the other components follow this convention.